src/mds/inode_backtrace.h

```diff
@@ -23,7 +23,8 @@ namespace ceph {
  */
 struct inode_backpointer_t {
   inode_backpointer_t() {}
-  inode_backpointer_t(inodeno_t i, std::string_view d, version_t v) : dirino(i), dname(d), version(v) {}
+  inode_backpointer_t(inodeno_t i, std::string_view d, version_t v, int a=-1) :
+    dirino(i), dname(d), version(v), is_auth(a) {}
```

Comment:
Am I missing something? The calls to this constructor (which need adjustment) are not changed in this PR.

Reply:
It is still in the stash. I have updated it.